_includes/election-summary.html

```diff
@@ -1,7 +1,18 @@
+{% assign max = 0 %}
+{% for contribution_type in election_totals['contributions_by_type'] %}
+  {% assign amount = contribution_type[1] %}
+  {% assign max = max | plus: amount %}
+{% endfor %}
+
+{% for source in election_totals.total_contributions_by_source %}
+  {% assign amount = contribution_type[1] %}
```

Comment:
Should the `amount` be assigned to the `source` variable that comes out of the for loop here? Or is this correct, in order to calculate/set the `max` variable?

Reply:
Looks like both of these loops could be removed as the max value is now provided at the key total_contributions.